vitrina/datasets/models.py

```diff
@@ -226,6 +226,13 @@ def tag_name_array(self):
     def category_title(self):
         return self.category.title if self.category else ""
 
+    def management_area(self):
```

Comment:
```suggestion
    def management_area(self) -> int | None:
```

Please, use type annotations.